packages/yoga/src/Button/web/Link.jsx

```diff
@@ -68,9 +68,14 @@ const Link = styled(Button)`
   }}
 `;
 
-const ButtonLink = ({ disabled, ...props }) => (
-  <Link {...props} disabled={disabled} aria-disabled={disabled} />
-);
+// eslint-disable-next-line react/prop-types
+const ButtonLink = forwardRef(({ isLoading, ...rest }, ref) => {
+  const { disabled } = rest;
+
+  return (
+    <Link {...rest} disabled={disabled} aria-disabled={disabled} ref={ref} />
+  );
+});
 
 ButtonLink.propTypes = {
```

Comment:
Shouldn't the isLoading be typed here?

Reply:
My intention here is to avoid using 'isLoading' when working with 'button.link' and 'button.text.' However, when I include the 'isLoading' types, it also adds this information to the yoga documentation, suggesting that 'isLoading' might be applicable.

Reply:
Got it, I didn't notice that these button types wouldn't have a loading state